test/csharp/provider-add-command/test.ts

```diff
@@ -0,0 +1,45 @@
+import { TestDriver, onPosix, onWindows } from "../../test-helper";
+
+describe("provider add command", () => {
+  let driver: TestDriver;
+
+  describe("pre-built", () => {
+    beforeAll(async () => {
+      driver = new TestDriver(__dirname, { CDKTF_DIST: "" }); // reset CDKTF_DIST set by run-against-dist script
+      await driver.setupCsharpProject({
+        init: { additionalOptions: "--cdktf-version 0.10.4" },
+      });
+    });
+
+    onPosix("installs pre-built provider using nuget", async () => {
```

Comment:
Why not run theses tests on Windows as well?

Reply:
No reason, added a todo for this to this PR.
I started with onPosix because I can create those snapshots locally, will add onWindows later on before merging this 👍